src/routes/assistant/[assistantId]/+page.server.ts

```diff
@@ -0,0 +1,11 @@
+import { collections } from "$lib/server/database.js";
+import { ObjectId } from "mongodb";
+
+export const load = async ({ params }) => {
+	const assistant = await collections.assistants.findOne({
```

Comment:
what happens if `assistant` is null. I.e. `assistant` with `params.assistantId` does not exist?

Reply:
added a check ! it redirects to the home settings now

Reply:
or is it guranteed to be non-null all the time?

Reply:
There is a check [here](https://github.com/huggingface/chat-ui/blob/f84c86a6bfba04137c88a6264233061d9842fad4/src/routes/settings/assistants/%5BassistantId%5D/%2Bpage.ts#L10) that redirects if it's null